dirs/dirs.go

```diff
@@ -255,6 +256,17 @@ func SnapBootAssetsDirUnder(rootdir string) string {
 	return filepath.Join(rootdir, snappyDir, "boot-assets")
 }
 
+// SnapDeviceDirUnder returns the path to device directory under rootdir.
+func SnapDeviceDirUnder(rootdir string) string {
```

Comment:
AFAICT this is not used in the code (the one below is) so we could not export it? But fine of course.